packages/web/components/Seeds/modals.tsx

```diff
@@ -0,0 +1,750 @@
+);
+
+export const FAQ = () => {
+  const [isOpenAccomplish, setIsOpenAccomplish] = useState(false);
+  const [isOpenInfinity, setIsOpenInfinity] = useState(false);
+  const [isOpenGrow, setIsOpenGrow] = useState(false);
+  const [isOpenCap, setIsOpenCap] = useState(false);
+
+  return (
+    <div>
+      <Box onClick={() => setIsOpenAccomplish(!isOpenAccomplish)} my={4}>
+        <details>
+          <summary>What have you accomplished so far?</summary>
+        </details>
+      </Box>
+      {isOpenAccomplish && (
+        <Text>
+          Many things! Go read the fundraise
+          <Link
+            ml={1}
+            isExternal
+            color="gray.500"
+            textDecoration="underline"
+            mr={1}
+            href="https://metagame.substack.com/p/calling-for-rain"
+          >
+            announcement post
+          </Link>
+          , pitch video or dig through the website itself.
+        </Text>
+      )}
+      <Box onClick={() => setIsOpenInfinity(!isOpenInfinity)} my={4}>
+        <details>
+          <summary>Will you just keep on minting Seeds to infinity?</summary>
+        </details>
+      </Box>
+      {isOpenInfinity && (
+        <Text>
+          Not likely! The idea is that MetaGame will go
+          <Link
+            ml={1}
+            isExternal
+            color="gray.500"
+            textDecoration="underline"
+            mr={1}
+            href="https://wiki.metagame.wtf/docs/how-does-it-work/phases-of-metagame"
+          >
+            through 3 phases
+          </Link>
+          ; starting with Seeds, growing into Trees in
+          <em>Phase II</em> aka <em>The Growth Phase </em>before transitioning
+          to Phase III - where Trees bear the fruits of our labor &amp; we’re
+          able to reward contributors with MetaGame’s incoming streams without
+          having to mint new tokens.
+        </Text>
+      )}
+      <Box onClick={() => setIsOpenGrow(!isOpenGrow)} my={4}>
+        <details>
+          <summary>Why should the price of Seeds grow?</summary>
+        </details>
+      </Box>
+      {isOpenGrow && (
+        <>
+          <Text>🌊 Seed Watering Streams</Text>
+          <UnorderedList mt={2}>
+            <ListItem>
+              all fees &amp; payments received by MetaGame go into sustaining
+              the price.
+            </ListItem>
+            <ListItem>
+              people subscribe to buying Seeds for any of the above reasons or
+              others.
+            </ListItem>
+            <ListItem>
+              projects we helped incubate share a % of their tokens?
+            </ListItem>
+          </UnorderedList>
+        </>
+      )}
+
+      <Box onClick={() => setIsOpenCap(!isOpenCap)} my={4}>
+        <details>
+          <summary>What’s market cap?</summary>
+        </details>
+      </Box>
+      {isOpenCap && (
+        <Text>
+          At the time of writing this post (February 2022), the market cap was
+          $600k
```

Comment:
Can we just link out to this calculation? Our market cap on mainnet is higher than this already

Reply:
The problem is that even the "real" calculation is wrong 😂
Displaying $861k at the price of $15.59 but the real price is more like $11..
![image](https://user-images.githubusercontent.com/55857222/154817088-8e5b2728-0545-4ba5-b026-4983dcb61768.png)

Reply:
can you gimme the link?

Reply:
https://etherscan.io/token/0x30cf203b48edaa42c3b4918e955fed26cd012a3f 

This is on mainnet however so it's not really relevant. Might as well just leave it as-is

Reply:
https://etherscan.io/token/0x30cf203b48edaa42c3b4918e955fed26cd012a3f
https://www.coingecko.com/en/coins/metagame